install-linux.sh

```diff
@@ -119,9 +119,37 @@ fi
 printf "\033[34m\n* Installing the Vanta Agent. You might be asked for your password...\n\033[0m"
 $SUDO $INSTALL_CMD $PKG_PATH
 
+##
+# Check whether the agent is registered. It may take a couple of seconds,
+# so try 3 times with 3-second pauses in between.
+##
+if [ -z "$VANTA_SKIP_REGISTRATION_CHECK" ] && [ -z "$VANTA_NOSTART" ]; then
+    printf "\033[34m\n* Checking registration with Vanta\n\033[0m"
+    registration_success=false
+    for i in {1..3}
+    do
+        echo "Attempt $i/3"
+        if $SUDO /var/vanta/vanta-cli check-registration; then
+            registration_success=true
+            break
+        fi
+        sleep 3
+    done
+
+    if [ "$registration_success" = false ] ; then
+        printf "\033[31m
+    Could not verify that the agent is registered to a Vanta domain. Are you sure you used the right key?
```

Comment:
Would be nice for this to explicitly indicate that the install is not complete.

Reply:
The install technically *is* complete; it just can't confirm that it's registered. It exits with status code 1.